cdc/owner/ddl_manager.go

```diff
@@ -184,6 +198,18 @@ func (m *ddlManager) tick(
 	checkpointTs model.Ts,
 	tableCheckpoint map[model.TableName]model.Ts,
 ) ([]model.TableID, *schedulepb.BarrierWithMinTs, error) {
+	// needBootstrap is true when the downstream is kafka
+	// and the protocol is simple protocol.
```

Comment:
I think this comment is redundant.

You can move it to the declaration of the `needBootstrap` variable

Reply:
After next PR merged, it will be remove.